.Rbuildignore

```diff
@@ -0,0 +1,29 @@
+# ignore JS config files/folders
+node_modules/
+coverage/
+src/
+lib/
+.babelrc
+.builderrc
+.eslintrc
+.npmignore
+
+# demo folder has special meaning in R
+# this should hopefully make it still
+# allow for the possibility to make R demos
+demo/.*\.js
+demo/.*\.html
+demo/.*\.css
+
+# ignore python files/folders
+setup.py
+usage.py
+setup.py
```

Comment:
```suggestion
config.py
```

Reply:
Since the R package makes the bioinformatics components available, but doesn't 'care' about the demo apps, should we ignore `index.py` as well?

Reply:
fixed in 2f15a26